src/scopes/grants/goalResponse.ts

```diff
@@ -0,0 +1,38 @@
+import { Op, WhereOptions } from 'sequelize';
+import { sequelize } from '../../models';
+
+function getSql(responses: string[]) {
+  return sequelize.literal(`(
+    WITH unnested_responses AS (
+      SELECT "goalId", unnest("response") AS res
+      FROM "GoalFieldResponses"
+    )
+    SELECT DISTINCT "Grants"."id"
+    FROM "Goals" "Goals"
+    INNER JOIN unnested_responses arr
+      ON arr."goalId" = "Goals"."id"
+    INNER JOIN "Grants" "Grants"
+      ON "Grants"."id" = "Goals"."grantId"
+    WHERE arr."res" IN (${responses.map((s) => `'${s}'`).join(', ')})
```

Comment:
Why join in the grant table when the grant id is on the goal?

Reply:
Thanks, fixed here: 8f4bf8b94